### PR TITLE
#1601 Remove System.Private

### PR DESCRIPTION
**Pull request by BrennanConroy**
@davidfowl

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -574,9 +574,6 @@ var CAN_BUILD_ONECORE = '${Directory.Exists(WIN10_SDK_LIB) && Directory.GetFiles
             {
                 var extraBinaries = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
                 {
-                    "System.Private.DataContractSerialization.dll",
-                    "System.Private.Networking.dll",
-                    "System.Private.ServiceModel.dll",
                     "System.Runtime.Serialization.Primitives.dll"
```
Comment (davidfowl) [inline makefile.shade]: Do we need this?

Comment (BrennanConroy) [inline makefile.shade]: Not sure, we added it because we broke some projects when we removed NewtonSoft: #1870

Comment (BrennanConroy) [inline makefile.shade]: Just built UserSecrets with the Primitives removed and it failed.

Comment (pranavkm) [inline makefile.shade]: Should we follow up with the serialization team? My guess is we don't want any of these binaries in the runtime if they aren't necessary.

Comment (davidfowl) [inline makefile.shade]: Ye we should nuke this.

**Comment by davidfowl** [comment]
:shipit:

**Comment by davidfowl** [comment]
@BrennanConroy What's the deal with this?

**Pull request merged**